blog/_posts/2017-05-22-https-on-stack-overflow.markdown

```diff
@@ -718,7 +718,7 @@ What may load just fine on the same domain on the same protocol may readily fail
 
 By having a CDN and app domains setup with the same protocols and layout we have in production, we find and fix many more issues before they leave a developer's machine.
 For example, did you know that when going from an `https://` page to an `http://` one, [the browser does not send the referer](https://www.w3.org/Protocols/rfc2616/rfc2616-sec15.html#sec15.1.3)?
-It's a security issue, there could be sensitive bits in the URL that would be sent over <span title="We saw this was a typo and opted to leave it.">paintext</span> in the referer header.
+It's a security issue, there could be sensitive bits in the URL that would be sent over <span title="We saw this was a typo and opted to leave it.">plaintext</span> in the referer header.
```

Comment:
Haha, totally missed the span. I just did a ctrl-f for "paintext". I do think it's a fun typo.